store/config.go

```diff
@@ -19,22 +20,23 @@ import (
 // Config holds parameters used by the application which can be overridden
 // by setting environment variables.
 type Config struct {
-	LogLevel               LogLevel `env:"LOG_LEVEL" envDefault:"info"`
-	RootDir                string   `env:"ROOT" envDefault:"~/.chainlink"`
-	Port                   string   `env:"CHAINLINK_PORT" envDefault:"6688"`
-	GuiPort                string   `env:"GUI_PORT" envDefault:"6689"`
-	BasicAuthUsername      string   `env:"USERNAME" envDefault:"chainlink"`
-	BasicAuthPassword      string   `env:"PASSWORD" envDefault:"twochains"`
-	EthereumURL            string   `env:"ETH_URL" envDefault:"ws://localhost:8546"`
-	ChainID                uint64   `env:"ETH_CHAIN_ID" envDefault:"0"`
-	ClientNodeURL          string   `env:"CLIENT_NODE_URL" envDefault:"http://localhost:6688"`
-	TxMinConfirmations     uint64   `env:"TX_MIN_CONFIRMATIONS" envDefault:"12"`
-	TaskMinConfirmations   uint64   `env:"TASK_MIN_CONFIRMATIONS" envDefault:"0"`
-	EthGasBumpThreshold    uint64   `env:"ETH_GAS_BUMP_THRESHOLD" envDefault:"12"`
-	EthGasBumpWei          big.Int  `env:"ETH_GAS_BUMP_WEI" envDefault:"5000000000"`
-	EthGasPriceDefault     big.Int  `env:"ETH_GAS_PRICE_DEFAULT" envDefault:"20000000000"`
-	LinkContractAddress    string   `env:"LINK_CONTRACT_ADDRESS" envDefault:"0x514910771AF9Ca656af840dff83E8264EcF986CA"`
-	MinimumContractPayment big.Int  `env:"MINIMUM_CONTRACT_PAYMENT" envDefault:"1000000000000000000"`
+	LogLevel                LogLevel        `env:"LOG_LEVEL" envDefault:"info"`
+	RootDir                 string          `env:"ROOT" envDefault:"~/.chainlink"`
+	Port                    string          `env:"CHAINLINK_PORT" envDefault:"6688"`
+	GuiPort                 string          `env:"GUI_PORT" envDefault:"6689"`
+	BasicAuthUsername       string          `env:"USERNAME" envDefault:"chainlink"`
+	BasicAuthPassword       string          `env:"PASSWORD" envDefault:"twochains"`
+	EthereumURL             string          `env:"ETH_URL" envDefault:"ws://localhost:8546"`
+	ChainID                 uint64          `env:"ETH_CHAIN_ID" envDefault:"0"`
+	ClientNodeURL           string          `env:"CLIENT_NODE_URL" envDefault:"http://localhost:6688"`
+	TxMinConfirmations      uint64          `env:"TX_MIN_CONFIRMATIONS" envDefault:"12"`
+	TaskMinConfirmations    uint64          `env:"TASK_MIN_CONFIRMATIONS" envDefault:"0"`
+	EthGasBumpThreshold     uint64          `env:"ETH_GAS_BUMP_THRESHOLD" envDefault:"12"`
+	EthGasBumpWei           big.Int         `env:"ETH_GAS_BUMP_WEI" envDefault:"5000000000"`
+	EthGasPriceDefault      big.Int         `env:"ETH_GAS_PRICE_DEFAULT" envDefault:"20000000000"`
+	LinkContractAddress     string          `env:"LINK_CONTRACT_ADDRESS" envDefault:"0x514910771AF9Ca656af840dff83E8264EcF986CA"`
+	MinimumContractPayment  big.Int         `env:"MINIMUM_CONTRACT_PAYMENT" envDefault:"1000000000000000000"`
+	SpecAndRunListenAddress *common.Address `env:"SPEC_AND_RUN_LISTEN_ADDR"`
```

Comment:
If this is essentially going to be the oracle contract address, it might be better to call it `OracleContractAddress` and `ORACLE_CONTRACT_ADDRESS`. Because we might also be able to use it for additional functionality later on.